.travis.yml

```diff
@@ -71,6 +74,7 @@ matrix:
       script:
           - cargo test -Z package-features --package netlink-proto --features audit
           - cargo test -Z package-features --package netlink-proto --features rtnetlink
+          - cargo test -Z package-features --package netlink-packet --features sock_diag
```

Comment:
Right now this fails because `netlink-proto` does not have this feature

Reply:
I added `sock_diag` feature to `netlink-proto` even its empty.